src/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rapidez\Core\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class SearchController
+{
+    public function __invoke(Request $request)
+    {
+        $query = Str::lower($request->q);
+        $searchQueryModel = config('rapidez.models.search_query');
+        if ($searchQuery = $searchQueryModel::firstOrCreate(
+            ['query_text' => $query],
+            ['store_id' => config('rapidez.store')]
+        )) {
```

Comment:
```suggestion
        if ($searchQuery = $searchQueryModel::firstOrCreate([
            'query_text' => $query,
            'store_id' => config('rapidez.store'),
        ])) {
```
The `store_id` should also be in the where right?